packages/xrpl/HISTORY.md

```diff
@@ -2,16 +2,20 @@
 
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
-## Unreleased Changes
+## Unreleased
```

Comment:
```suggestion
## Unreleased Changes
```

Can we retain the words here? I feel the existing wording is better.

Reply:
Every other package uses "Unreleased", and thats whats consistent with https://keepachangelog.com/en/1.1.0/ so thats why I made the change. If you feel strongly, I can keep it but felt inconsistent

Reply:
Okay, I wasn't aware of this industry practice.